library/NotificationCenter/Gateway/Email.php

```diff
@@ -161,7 +161,7 @@ public function sendDraft(EmailMessageDraft $objDraft)
         try {
             return $objEmail->sendTo($objDraft->getRecipientEmails());
         } catch (\Exception $e) {
-            \System::log(sprintf('Could not send email for message ID %s: %s', $objDraft->getMessage()->id, $e->getMessage()), __METHOD__, TL_ERROR);
+            \System::log(sprintf('Could not send email for message ID %s: %s, %s', $objDraft->getMessage()->id, $e->getMessage(), implode(',',$objDraft->getRecipientEmails())), __METHOD__, TL_ERROR);
```

Comment:
```suggestion
            \System::log(sprintf('Could not send email to "%s" for message ID %s: %s', implode(', ', $objDraft->getRecipientEmails())), $objDraft->getMessage()->id, $e->getMessage(), __METHOD__, TL_ERROR);
```

Reply:
@zonky2 I am not sure this change is actually necessary though? In case `Email::sendTo` fails, Contao will log

```
E-mail address rejected: foo@example.com
```

anyway, will it not?

Reply:
fixed